### ISSUE TITLE
subscriberScreen overlap publisherScreen 

### ISSUE THREAD

**Issue by hero70312** [opentok/opentok-react-native#156]
We found some android device's publisherScreen disappear.
So we minify the subscriberScreen size, then we found that subscriberScreen overlap publisherScreen.

Left side mobile screen is android 8.0.
Right side mobile screen is iPhoneX 

<img width="996" alt="screenshot" src="https://user-images.githubusercontent.com/14068699/44252545-14260a00-a22f-11e8-9404-7fba67676c02.png">

**Comment by msach22** [comment]
@hero70312 Thanks for reporting this. Can you please share the stylesheets of how you're rendering the components?

**Comment by hero70312** [comment]
Sure!

These are my stylesheets. 

```
let {height, width} = Dimensions.get('window');
```

```
 render() {
        const {user, chatRoom} = this.props;
        const token = (user.user_type === UserType.VIP) ? chatRoom.token_vip : chatRoom.token_doctor;
        return (
            <SafeAreaView style={{flex: 1, ...CommonStyles.color.bg.black}}>
                    <OTSession apiKey={Config.opentokApiKey} sessionId={chatRoom.session_id} token= 
                    {token} style={styles.container}>
                        <View style={styles.subscriberContainer}>
                             <OTSubscriber style={{ width, height}} />
                        </View>
                        <View style={styles.publisherContainer}>
                             <OTPublisher style={{ width: 100, height: 100 }} />
                        </View>
                    </OTSession>
            </SafeAreaView>
        );
    }
```

```
    subscriberContainer: {
        flex: 1,
    },
    publisherContainer: {
        position: 'absolute',
        right: 0,
        paddingTop: 10,
        paddingRight: 10,
    },
```

**Comment by msach22** [comment]
@hero70312 I believe this is a duplicate of https://github.com/opentok/opentok-react-native/issues/122 which as been fixed and release with `v0.8.4`.
